OPCB/functions/missions/gdrunken_spawn.sqf

```diff
@@ -1,38 +1,37 @@
 params ["_pos"];
-_comps = [];
-_group = createGroup east;
-_tanks = [];
+private _comps = [];
+private _tanks = [];
 
 for "_i" from 0 to 3 do {
-	_suitable = [0, 0, 0];
+	_suitable = [0, 0, 0, 0];
 	_maxDist = 1500;
 	_list = [];
 
-	while {((count _suitable) == 3) || {(count _list) > 0}} do {
+	while {count _suitable == 4 || {surfaceIsWater _suitable}} do {
```

Comment:
Why did you change this to 4? If it fails, it will be a 3 element array, which will default to the map centre, which may or may not be on water. If it succeeds it will be 2. This is wrong.

Reply:
this is from my previous tests. This works, count 3 also works, doesn't matter honestly. 
quote: Result Array -  in format [x,y] on success. When position cannot be found at all, default map center position is returned, which will be in format [x,y,0]. 
Which means that if it fails, it's count == 3 and runs again, so it doesn't matter if it's 4 or 3 dim for suitable locations.

Reply:
What? How does it run again if it's count==3 with this code?

Reply:
because I checked and all of the map default positions are water, so the 2nd check starts the loop again

Reply:
Ok but what about other maps?!

Reply:
I checked the maps in the repo folder. Future maps might differ.

Reply:
This is going to end up being a miserable tech debt in the future that I hope you remember.